numpy/ma/core.py

```diff
@@ -6094,6 +6094,22 @@ def take(self, indices, axis=None, out=None, mode='raise'):
     T = property(fget=lambda self: self.transpose())
     transpose = _arraymethod('transpose')
 
+    @property
+    def mT(self):
+        """
+        TODO: Fill in documentation
+        """
+
+        if self.ndim < 2:
+            raise ValueError("matrix transpose with ndim < 2 is undefined")
+
+        if self._mask is nomask:
+            # TODO: Should this return a masked_array or regular array?
+            return self._data.mT
```

Comment:
A masked array.  Otherwise, seems good.  Although `.transpose()` has some `_update_from_self` stuff, it may make sense to just stick close to it in the off-chance it matters.

Reply:
I'm struggling to replicate `.transpose()` as it uses the function version, not the attribute. I haven't been figured out how to do this (yet) and will leave it for another PR - maybe I can come back and tidy it up then if there are problems?